thread.c

```diff
@@ -1873,6 +1873,9 @@ rb_thread_io_blocking_region(rb_blocking_function_t *func, void *data1, int fd)
  *       created as Ruby thread (created by Thread.new or so).  In other
  *       words, this function *DOES NOT* associate or convert a NON-Ruby
  *       thread to a Ruby thread.
+ *
+ * NOTE: If this thread has already aquired the GVL, then the method call
```

Comment:
👋 Hello! I noticed a minor typo: `aquired` -> `acquired`

Reply:
thank you @alexcameron89